07_boolean_2/main.go

```diff
@@ -15,56 +15,128 @@ E.g.:
 */
 
 func main() {
-	fmt.Println(false && (false || true))
-	fmt.Println((false && false) || true)
+	// fmt.Println(false && (false || true))
+	// fmt.Println((false && false) || true)
+	logicalTable(true, true)
+	logicalTable(true, false)
+	logicalTable(false, false)
+	logicalTable(false, true)
+	strIntCheck("", 0)
+	strIntCheck("Gopher", 3)
+	strIntCheck("Git", 2)
+	strIntCheck("Go", 4)
+	// gradeValid(-10, 101, 101, 101, 90)
+}
 
-	// TODO: Write a function that prints a truth table for all the following
-	// Operations:
-	//  - AND
-	//  - OR
-	//  - NAND (NOT AND)
-	//  - NOR  (NOT OR)
-	//  - XOR  (EXCLUSIVE OR)
-	// E.g.: logicalTable(true, true) would print:
-	// AND : true
-	// OR  : true
-	// NAND: false
-	// ...
-	// Call this function for all 4 combinations of true and false.
+// TODO: Write a function that prints a truth table for all the following
+// Operations:
+//   - AND
+//   - OR
+//   - NAND (NOT AND)
+//   - NOR  (NOT OR)
+//   - XOR  (EXCLUSIVE OR)
+//
+// E.g.: logicalTable(true, true) would print:
+// AND : true
+// OR  : true
+// NAND: false
+// ...
+// Call this function for all 4 combinations of true and false.
+func logicalTable(bool1 bool, bool2 bool) {
+	// Varible used
+	// var AND bool = bool1 && bool2
+	// fmt.Println(AND)
+	// Hardcoded print results
+	// if bool1 == true && bool2 == true {
+	// 	fmpt.Println("true")
+	// }
+	// Direct print statements
+	// AND // true, true == true || true, false == false || false, false == false || false, true == false
+	fmt.Println("AND", bool1 && bool2)
+	// OR // true, true == true || true, false == true || false, false == false || false, true == true
+	fmt.Println("OR", bool1 || bool2)
+	// NAND // true, true == false || true, false == true, || false, false == true || false, true == true
+	fmt.Println("NAND", !bool1 || !bool2)
```

Comment:
Adding an alternative
```suggestion
	fmt.Println("NAND", !bool1 || !bool2)
	fmt.Println("NAND", !(bool1 && bool2))
```

Reply:
Added, thanks!